root/static/styles/entity-icons.less

```diff
@@ -8,7 +8,13 @@
 }
 
 .blankheader      { background-image: data-uri('../images/entity/blank.svg') }
-.artistheader     { background-image: data-uri('../images/entity/artist.svg') }
+.artistheader     { &.artistico     { background-image: data-uri('../images/entity/artist.svg') }
```

Comment:
`artist.svg` should be available as a fallback, that is, the default `background-image` for the class `artistheader`. The class `artistico` would be unnecessary then and should be removed. That would also be used for artist with either no type or `other` type (or any further unknown type). The rest of changes to this file is fine. 👍